curriculum/challenges/english/02-javascript-algorithms-and-data-structures/basic-data-structures/add-key-value-pairs-to-javascript-objects.md

```diff
@@ -81,13 +81,12 @@ The `foods` object should have a key `strawberries` with a value of `27`.
 assert(foods.strawberries === 27);
 ```
 
-The key-value pairs should be set using dot or bracket notation.
+The `foods` object's first definition should not be changed directly.
```

Comment:
This feels a tiny bit awkward to my ear. Maybe
```suggestion
The definition of the `foods` object should not be changed.
```

Reply:
I agree that sounds smoother. Thank you.